sdkconfig

```diff
@@ -322,6 +322,7 @@ CONFIG_ESP32S2_TRACEMEM_RESERVE_DRAM=0x0
 # CONFIG_ESP32S2_ULP_COPROC_ENABLED is not set
```

Comment:
Can you revert his file please? It shouldn't have changed, but I guess the IDF tweaked it during a cmake build or something

Reply:
Fixed